tests/broad/scala_test/src/main/scala/org/broadinstitute/dsp/pipelines/tester/RNAWithUMIsTester.scala

```diff
@@ -8,11 +8,11 @@ import better.files.File
 import org.broadinstitute.dsp.pipelines.batch.WorkflowTest
```

Comment:
The error message in the smart tests is strange (none of the arguments to the tes framework appear invalid to me), but you will need to update the name of the pipeline on line 20 in this file